projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -576,6 +575,13 @@ public function update_settings() {
 						$updated[ $key ] = $value;
 					}
 					break;
+				case 'launchpad_screen':
+					if ( in_array( $value, array( 'full', 'off', 'minimized' ), true ) ) {
```

Comment:
Hey @jeyip 👋🏼  I see this section was introduced in D85918-code originally. Would you say it would be alright to switch `in_array()` to use strict comparison - in contrast to your original code (in the patch above)?

Thank you!

Background: In order to make the Jetpack files in sync with its WPCOM counterpart, we need to make a couple of changes to make linters happy.

Reply:
Related D89665-code is now ready for review.